rbac/providers/common/admin_setup.py

```diff
@@ -21,16 +21,43 @@
 
 from environs import Env
 
+from rbac.providers.common.common import wait_for_rethink
+from tests.utilities import wait_for_resource_in_db
+
 LOGGER = logging.getLogger(__name__)
```

Comment:
We should now import our global get_default_logger here to help us print out some of the logging statements in this file.